src/app/routes/api/v3/forms/__tests__/public-forms.auth.routes.spec.ts

```diff
@@ -53,7 +54,12 @@ describe('public-form.auth.routes', () => {
       // Assert
       expect(response.status).toEqual(StatusCodes.OK)
       expect(response.body).toMatchObject({
-        redirectURL: expect.any(String),
+        redirectURL: expect.stringContaining(
+          encodeURI(getRedirectTarget(form._id, AuthType.SP, false)),
+        ),
+      })
+      expect(response.body).toMatchObject({
+        redirectURL: expect.stringContaining(form.esrvcId!),
```

Comment:
nit:  can be refactored to check everything in 1 block:

```ts
expect(response.body).toMatchObject({
  redirectUrl: expect.toIncludeMultiple([
    encodeURI(getRedirectTarget(form._id, AuthType.SP, false)), 
    form.esrvcId!
  ])
})
```

Reply:
ah good one. wasn't aware of this because i was looking through jest docs and they didn't have this oddly enough